### PR TITLE
Flake8 the project

### PR DESCRIPTION
**Pull request by KevinEtienne**

### CODE REVIEW

**Comment by KevinEtienne** [comment]
@NoLogo @adam-incuna can you review/merge please?

**Comment by adam-incuna** [comment]
Nice one :) :+1:

File: Makefile [hunk 1]
```diff
@@ -9,8 +9,6 @@ release:
 	python setup.py register bdist_wheel upload
 
 test:
-	python -Wall tests/run.py
```
Comment (Ian-Foote) [inline Makefile]: The `-Wall` would have been useful to keep around.

Comment (KevinEtienne) [inline Makefile]: Sounds good I'll add it back to the Makefile under another name

**Pull request merged**